### PR TITLE
build: fix dev process on windows

### PR DESCRIPTION
**Pull request by elevatebart**

### CODE REVIEW

File: scripts/gulp/utils/childProcessUtils.ts [hunk 1]
```diff
@@ -269,3 +265,14 @@ function streamHandler (cp: ChildProcess, config: StreamHandlerConfig) {
...
+const isWin = process.platform === 'win32'
+
+/**
+ * Pretreat commands to make them compatible with windows
+ * @param command
+ * @returns
+ */
+export function winSpawn (command: string): string {
+  return `${(command).replace(/\//g, '\\')}${isWin ? '.cmd' : ''}`
+}
```
Comment (baus) [inline scripts/gulp/utils/childProcessUtils.ts:278]: This seems like a strange name since it works on *nix and Windows

Comment (elevatebart) [reply scripts/gulp/utils/childProcessUtils.ts:278]: I agree `universalSpawn` woudl be better

**Comment by elevatebart** [comment]
@modern-sapien see this one

**Review by lmiller1990 — COMMENTED** [review]
just confirming - does the linux/macOS still work after this change? did you test that, too?

**Comment by elevatebart** [comment]
@lmiller1990 I checked macos. Linux should be checked by CI but I cannot vouch for the development environment.

DO you think it is worth dusting off the old docker?

**Comment by lmiller1990** [comment]
Should be good, macOS and linux are the same for `/` in paths.

**Pull request merged**